src/jobs/veracode_pipeline_scan.yml

```diff
@@ -13,14 +13,11 @@ parameters:
     description: Directory inside the docker image where the application artifacts are saved
     type: string
     default: "/app"
+docker:
+  - image: 'veracode/pipeline-scan:latest'
 steps:
   - veracode_prepare_artifacts:
       docker_image_app_dir: << parameters.docker_image_app_dir >>
-  - run:
-      name: Download/Extract-Veracode-Pipeline-Scanner
-      command: |
-        wget https://downloads.veracode.com/securityscan/pipeline-scan-LATEST.zip -O pipeline-scan.zip
```

Comment:
While this does get `pipeline-scan-LATEST.zip` I thought it made sense to be consistent so also sourcing from Docker Hub